snapcraft/cli/assertions.py

```diff
@@ -13,10 +13,34 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import os
+import subprocess
+import tempfile
+from datetime import datetime
+from textwrap import dedent
 
 import click
+import yaml
 
 import snapcraft
+from . import echo
+from snapcraft import storeapi
+from snapcraft.storeapi import assertions
+
+
+_COLLABORATION_HEADER = dedent("""\
+    # Change which developers may push or release snaps on the publisher's behalf.
+    #
+    # Sample entry:
+    #
+    # developers:
+    #   - developer-id: "dev-one"      # Which developer
+    #     since: "2017-02-10 08:35:00" # When contributions started
+    #     until: "2018-02-10 08:35:00" # When contributions ceased (optional)
+    #
+    # All timestamps are UTC, and the "now" special string will be replaced by
+    # the current time. Do not remove entries or use an until time in the past
+    # unless you want existing snaps provided by the developer to stop working.""")  # noqa
```

Comment:
"stop working" is interesting. It might be that something continues to work but cannot be installed. However, I'm not sure there's a better description until things start using snap-developer.